//
//  exception.cpp
//  code
//
//  Created by 김도현 on 2023/04/02.
//

#include <iostream>
using namespace std;

try
{
    ...........
    if(예외가 발생한다면)
        throw expn;
    ..............
}
catch(type expn)
{
    예외처리
}

반드시 try 다음에는 반드시 catch !!
